### DISCUSSION TITLE
Github Education keeps denying me (25 trys)

### DISCUSSION THREAD

**Discussion by TheoH32** [community/community#60469] [GitHub Education]
### Hi everyone,

First off, I applied and got into GitHub Education in middle school. Now that it expired, I keep getting denied for re-application and can't figure out why. I edited my profile to include a lot about me, I've tried different documents, etc. Every time I try to submit it, it shows up as denied instantly. The only thing I can think of is that I am using a personal email (which I used to get in the first time) because my school doesn't allow us to access our school email inbox meaning I can add my school email to my account but can't verify it. I also tried opening a ticket a month ago, but no one from support has responded still. If someone can give me tips or if someone from support can help, that would mean a lot. Thanks.
<img width="1512" alt="Screenshot 2023-07-09 at 7 59 00 PM" src="https://github.com/community/community/assets/47646488/c9a8355c-9105-4a38-a1f7-a57b01af0471">

**Comment by D7x7z49** [comment]
You can view this question: <https://github.com/orgs/community/discussions/59360>

Pay attention to this issue to increase its weight
